From 20a854f3e81d26b6f40b3b16e6277563cc14200f Mon Sep 17 00:00:00 2001
From: Ido Schimmel <idosch@nvidia.com>
Date: Wed, 30 Apr 2025 13:02:40 +0300
Subject: [PATCH 02/30] net: phy: maxio: Fix disable EEE

phydev->eee_broken_modes has been replaced by a helper.

The code now compiles :-)

Signed-off-by: Andrew Lunn <andrew@lunn.ch>
---
 drivers/net/phy/maxio.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/phy/maxio.c b/drivers/net/phy/maxio.c
index 260a89d95..125358d83 100755
--- a/drivers/net/phy/maxio.c
+++ b/drivers/net/phy/maxio.c
@@ -154,7 +154,7 @@ static int maxio_mae0621a_config_init(struct phy_device *phydev)
 	printk("eee value: 0x%x \n",maxio_read_mmd(phydev, MDIO_MMD_AN, MDIO_AN_EEE_ADV));
 	broken |= MDIO_EEE_100TX;
 	broken |= MDIO_EEE_1000T;
-	phydev->eee_broken_modes = broken;
+	phy_disable_eee(phydev);
 
 	//enable auto_speed_down
 	ret = maxio_write_paged(phydev, 0xd8f, 0x0, 0x300 );
-- 
2.39.5

